### PR TITLE
Add paths for downloaded files to static quantization tutorial

### PR DESCRIPTION
**Pull request by SethHWeidman**
@raghuramank100 adding language making it clear for readers how to download the files associated with this tutorial.

### CODE REVIEW

**Comment by SethHWeidman** [comment]
@raghuramank100 merging!

Closes PyTorch issue #28844.

File: advanced_source/static_quantization_tutorial.py [hunk 1]
```diff
@@ -348,7 +374,8 @@ def prepare_data_loaders(data_path):
     return data_loader, data_loader_test
 
 ######################################################################
-# Next, we'll load in the pre-trained MobileNetV2 model
+# Next, we'll load in the pre-trained MobileNetV2 model. Similarly to the data about, the file with the pre-trained
+# weights is stored at ``https://s3.amazonaws.com/pytorch-tutorial-assets/mobilenet_quantization.pth``:
```
Comment (raghuramank100) [inline advanced_source/static_quantization_tutorial.py]: This is confusing to end users. Let us use the file from torchvision at: https://github.com/pytorch/vision/blob/master/torchvision/models/mobilenet.py#L9

Also, lets call this file something else: not mobilenet_quantization.pth, might be better to call it mobilenet_pretrained_float.pth'

File: advanced_source/static_quantization_tutorial.py [hunk 2]
```diff
@@ -306,7 +306,33 @@ def print_size_of_model(model):
 # As our last major setup step, we define our dataloaders for our training and testing set.
 # The specific dataset we've created for this tutorial contains just 1000 images, one from
 # each class (this dataset, at just over 250 MB, is small enough that it can be downloaded
-# relatively easily). These functions mostly come from
+# relatively easily). The URL for this custom dataset is:
```
Comment (raghuramank100) [inline advanced_source/static_quantization_tutorial.py]: We should also add information on how one can download imagenet for pytorch here. We could say that:
If you want to download the entire imagenet dataset, please follow instructions at:

https://pytorch.org/docs/stable/torchvision/datasets.html#imagenet

**Comment by SethHWeidman** [comment]
@raghuramank100 tried to address these changes, see [here](https://deploy-preview-713--pytorch-tutorials-preview.netlify.com/advanced/static_quantization_tutorial.html)

**Comment by z-a-f** [comment]
@SethHWeidman There is an issue with showing the tutorial version (#28938), which means the users will not see it until the next major release.. Is there a way to force this change to either the current branch or to the 1.3.1?

**Comment by SethHWeidman** [comment]
@z-a-f sorry, users will not see _what_ until the next major release? The tutorial version? Meaning if we update the version to 1.3.1, it will still show as 1.3.0?

In case there's any confusion, these changes are live; see [here](https://pytorch.org/tutorials/advanced/static_quantization_tutorial.html).

Help me understand what the issue is and what you think we should do. Thanks!

**Pull request merged**